packages/foam-vscode/src/features/tags-tree-view/index.spec.ts

```diff
@@ -68,7 +68,6 @@ describe('Tags tree panel', () => {
     const parentTreeItems = (await provider.getChildren()) as Tag[];
     const parentTagItem = parentTreeItems.pop();
     expect(parentTagItem.title).toEqual('parent');
-    expect(parentTagItem.title).not.toEqual('child');
```

Comment:
a comment about line 34 of this file, where you initialize `_foam`. 
I feel this pattern (which IIRC you picked up from`backlinks.spec.ts` might not apply very well here.
The reason is that in that file, `foam` was used for read only.
In these tests you are changing the state of the `_foam` object and this makes things harder to track.
I would suggest one of these approaches:

1. move the foam bootstrap (and disposal) into every test
2. move the foam bootstrap into `beforeEach` and disposal into `afterEach`

Of course happy to hear other approach if you prefer a different take.

This will allow in each test to check exactly the state you would expect, without risk of extra objects polluting your test workspace.

Thoughts?